eng/Tools.props

```diff
@@ -1,14 +1,7 @@
 <Project>
   <PropertyGroup>
-    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
-      https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json;
```

Comment:
Do these have to be added to the repo `NuGet.config`?

Reply:
No, the dotnet-buildtools one is auto added by arcade and the sourcelink one isn't necessary anymore.